x-pack/test/fleet_api_integration/config.ts

```diff
@@ -15,7 +15,7 @@ import { defineDockerServersConfig } from '@kbn/test';
 // example: https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fpackage-storage/detail/snapshot/74/pipeline/257#step-302-log-1.
 // It should be updated any time there is a new Docker image published for the Snapshot Distribution of the Package Registry.
 export const dockerImage =
-  'docker.elastic.co/package-registry/distribution:b6a53ac9300333a4a45f3f7d350c9aed72061a66';
+  'docker.elastic.co/package-registry/distribution@sha256:35cedaaa6adac547947321fa0c3b60a63eba153ba09524b9c1a21f1247a09bd2';
```

Comment:
cc @jfsiii for awareness that this change may overlap with https://github.com/elastic/kibana/pull/101911

Reply:
thanks, @jen-huang. @rw-access am I correct that this only needs this SHA for now?  i.e. after this lands or, say, 2 weeks from now, we should be able to update using the current approach of `distribution:some_recent_tag`?

Reply:
yeah, I just needed a SHA that was at least this recent.
should this be switched to a tag?

Reply:
that sound good to you both @jen-huang and @jfsiii?
please let me know if there are any changes I need to make

Reply:
@rw-access sorry for the lag, I missed this notification.

I'm fine with leaving as a SHA. After it lands, we can return to a `distribution:*` tag.